src/main/java/com/softserveinc/ita/rozetka/page_objects/OrderPage.java

```diff
@@ -1,4 +1,18 @@
 package com.softserveinc.ita.rozetka.page_objects;
 
+import static com.codeborne.selenide.Selenide.$x;
+
 public class OrderPage {
+    public OrderPage fillAllInputFields(){
+        $x("//div[@class='form__row js-surname']/input").setValue("Петренко");
+        $x("//div[@class='form__row js-name']/input").setValue("Вася");
+        $x("//div[@class='autocomplete']/input").setValue("Івано-Франківськ").click();
+        $x("//div[@class='form__row js-phone']/input").setValue("77777777");
```

Comment:
do not hard code, do not mix behavior
re-do into
setName()
setSurname()
setPhone()

Reply:
setShippingAddress(address) {
setName(); setSurname();
}